src/styles/common/_buttons.scss

```diff
@@ -1,35 +1,87 @@
+@import '../constants/typography';
+@import '../constants/layout';
+@import '../mixins/buttons';
+
 /**************************************
  * Buttons
  **************************************/
+$bdl-btn-border-width: 1px;
+$bdl-btn-height: 32px;
+$bdl-btn-height-large: 40px;
```

Comment:
Should we use SUIT for our variable names here, something like `$bdl-Button-height--large`?